Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/SingleCardView.swift

```diff
@@ -0,0 +1,241 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+import Storage
+import Common
+
+import Foundation
+import UIKit
+
+class SingleCreditCardViewController: UIViewController, BottomSheetChild {
+    // MARK: UX
+    struct SingleCardViewControllerUX {
+        static let tableMargin: CGFloat = 0
+        static let distanceBetweenHeaderAndTop: CGFloat = -10
+        static let distanceBetweenButtonAndTable: CGFloat = 77
+        static let distanceBetweenHeaderAndCells: CGFloat = 24
+        static let yesButtonCornerRadius: CGFloat = 13
+        static let yesButtonFontSize: CGFloat = 16.0
+        static let yesButtonHeight: CGFloat = 45.0
+        static let bottomSpacing: CGFloat = 32.0
+        static let buttonsSpacing: CGFloat = 8.0
+        static let contentViewWidth: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 545 : 339
+        static let headerPrefferredHeight: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 64 + 24 : 84 + 24
+        static let estimatedRowHeight: CGFloat = 86
+    }
+    var notificationCenter: NotificationProtocol
+    var themeManager: ThemeManager
+    var themeObserver: NSObjectProtocol?
+    var creditCard: CreditCard
```

Comment:
nit: do you need access to creditCard, can we make it private?

```suggestion
    private var creditCard: CreditCard
```